--- DevIL-1.7.8/devil-1.7.8/configure.ac~	2009-03-08 21:43:03.000000000 +0100
+++ DevIL-1.7.8/devil-1.7.8/configure.ac	2010-04-21 15:40:09.671716250 +0200
@@ -154,6 +154,17 @@
 AS_IF([test  "x$restrict_keyword" = "xfound"],
       [DEVIL_CHECK_RESTRICT_GNU99])
 
+dnl checking if the -fgnu89-inline flag is available
+TMP_CFLAGS="$CFLAGS"
+CFLAGS="$CFLAGS -fgnu89-inline"
+AC_MSG_CHECKING([gnu89-inline option])
+AC_COMPILE_IFELSE([[void main(void){}]],
+	[AC_MSG_RESULT(yes)]
+	[CFLAGS="$TMP_CFLAGS"]
+	[ADD_CFLAGS(-fgnu89-inline)]
+	,[AC_MSG_RESULT(no)]
+	[CFLAGS="$TMP_CFLAGS"])
+
 dnl must add AC_CACHE_CHECK support where possible
 
 dnl AM_MAINTAINER_MODE
