--- DevIL-1.6.7/src-ILU/src/Makefile.am.orig	2004-06-22 18:47:25.000000000 +0200
+++ DevIL-1.6.7/src-ILU/src/Makefile.am	2006-04-16 20:57:44.000000000 +0200
@@ -9,8 +9,7 @@
 
 AM_CFLAGS = @LIBILU_CFLAGS@
 
-libILU_la_LDFLAGS = -no-undefined @VERSIONINFO@ \
-                    -L$(top_srcdir)/src-IL/src/
-libILU_la_LIBADD = -lIL
+libILU_la_LDFLAGS = -no-undefined @VERSIONINFO@
+libILU_la_LIBADD = $(top_builddir)/src-IL/src/libIL.la
 
 INCLUDES = -I$(top_srcdir)/include -I$(top_srcdir)/src-ILU/include
--- DevIL-1.6.7/src-ILUT/src/Makefile.am.orig	2004-06-22 18:47:26.000000000 +0200
+++ DevIL-1.6.7/src-ILUT/src/Makefile.am	2006-04-16 21:02:00.000000000 +0200
@@ -9,11 +9,9 @@
 AM_CFLAGS = @LIBILUT_CFLAGS@
 
 libILUT_la_LDFLAGS = -no-undefined @VERSIONINFO@ \
-                     -L$(top_srcdir)/src-IL/src \
-                     -L$(top_srcdir)/src-ILU/src \
                      @LIBSDL@ @LIBALLEGRO@ @LIBILUT_WIN32LIBPATH@
 
-libILUT_la_LIBADD = -lIL -lILU @LIBGL@ @LIBGDI32@
+libILUT_la_LIBADD = $(top_builddir)/src-IL/src/libIL.la $(top_builddir)/src-ILU/src/libILU.la @LIBGL@ @LIBGDI32@
 
 INCLUDES = -I$(top_srcdir)/include -I$(top_srcdir)/src-ILUT/include \
            @HSDL@ @HALLEGRO@ @X_CFLAGS@
