--- DevIL-1.7.2/configure.ac.orig	2008-09-05 03:19:19.000000000 +0200
+++ DevIL-1.7.2/configure.ac	2008-10-22 18:13:22.182086647 +0200
@@ -191,27 +191,6 @@
 dnl Special code for libjpeg
 AC_DEFINE(IL_USE_JPEGLIB_UNMODIFIED,,[Use libjpeg without modification. always enabled.])
 
-dnl
-dnl Setup Libraries
-dnl
-AC_DEFUN([SET_LIB],[
-LIB$2=""
-AC_ARG_WITH($1,AC_HELP_STRING([--with-$1=path],[Specify the lib$1 installation path]),
-	ADD_CFLAGS([-I$enable_$1dir/include])
-	LIB$2="-L$enable_$1dir/lib"
-	no_$1=no
-	use_$1=yes )
-AS_IF([test ! "$no_$1" = "yes" && test "$use_$1" = "yes" && test -z "$3"],[LIB$2="$LIB$2 -l$1"]
-     ,[test ! "$no_$1" = "yes" && test "$use_$1" = "yes" && ! test -z "$3"],[LIB$2="$3"],[])
-AC_SUBST(LIB$2) ])
-
-SET_LIB(jpeg,JPG)
-SET_LIB(tiff,TIFF)
-SET_LIB(lcms,LCMS)
-SET_LIB(mng,MNG)
-SET_LIB(jp2,JP2)
-SET_LIB(z,Z)
-
 dnl libpng configuration
 AC_MSG_CHECKING(libpng)
 LIBPNG_INCLUDE=""
@@ -278,6 +257,27 @@
 AC_SUBST(LIBPNG_INCLUDE)
 
 dnl
+dnl Setup Libraries
+dnl
+AC_DEFUN([SET_LIB],[
+LIB$2=""
+AC_ARG_WITH($1,AC_HELP_STRING([--with-$1=path],[Specify the lib$1 installation path]),
+	ADD_CFLAGS([-I$enable_$1dir/include])
+	LIB$2="-L$enable_$1dir/lib"
+	no_$1=no
+	use_$1=yes )
+AS_IF([test ! "$no_$1" = "yes" && test "$use_$1" = "yes" && test -z "$3"],[LIB$2="$LIB$2 -l$1"]
+     ,[test ! "$no_$1" = "yes" && test "$use_$1" = "yes" && ! test -z "$3"],[LIB$2="$3"],[])
+AC_SUBST(LIB$2) ])
+
+SET_LIB(jpeg,JPG)
+SET_LIB(tiff,TIFF)
+SET_LIB(lcms,LCMS)
+SET_LIB(mng,MNG)
+SET_LIB(jp2,JP2,[-ljasper])
+SET_LIB(z,Z)
+
+dnl
 dnl ILUT APIs checking
 dnl
 AC_DEFUN([TEST_API],[
--- DevIL-1.7.2/src-IL/src/Makefile.am.orig	2008-08-24 19:32:38.000000000 +0200
+++ DevIL-1.7.2/src-IL/src/Makefile.am	2008-10-22 18:22:57.715154030 +0200
@@ -15,6 +15,6 @@
 
 AM_CFLAGS = @LIBIL_CFLAGS@
 libIL_la_LDFLAGS = @WIN32_EXTRA_LDFLAGS@ @VERSIONINFO@
-libIL_la_LIBADD = @LIBPNG@ @LIBJPG@ @LIBTIFF@ @LIBMNG@ @LIBLCMS@ @LIBZ@
+libIL_la_LIBADD = @LIBPNG@ @LIBJPG@ @LIBTIFF@ @LIBMNG@ @LIBLCMS@ @LIBZ@ @LIBJP2@
 
 INCLUDES = -I$(top_srcdir)/include -I$(top_srcdir)/src-IL/include -I$(top_builddir)/include @LIBPNG_INCLUDE@
