--- DevIL-1.7.2/src-IL/src/il_png.c.orig	2010-02-07 16:59:29.208824733 +0100
+++ DevIL-1.7.2/src-IL/src/il_png.c	2010-02-07 17:04:17.675411333 +0100
@@ -103,7 +103,11 @@ ILboolean iIsValidPng()
 	Read = iread(Signature, 1, 8);
 	iseek(-Read, IL_SEEK_CUR);
 
+#if PNG_LIBPNG_VER < 10400
 	return png_check_sig(Signature, 8);
+#else
+	return png_sig_cmp(Signature, 0, 8) == 0;
+#endif
 }
 
 
@@ -278,7 +282,11 @@ ILboolean readpng_get_image(ILdouble dis
 
 	// Expand low-bit-depth grayscale images to 8 bits
 	if (color_type == PNG_COLOR_TYPE_GRAY && bit_depth < 8) {
+#if PNG_LIBPNG_VER < 10400
 		png_set_gray_1_2_4_to_8(png_ptr);
+#else
+		png_set_expand_gray_1_2_4_to_8(png_ptr);
+#endif
 	}
 
 	// Expand RGB images with transparency to full alpha channels
--- DevIL-1.7.2/src-IL/src/il_icon.c.orig	2010-02-07 17:15:17.456176881 +0100
+++ DevIL-1.7.2/src-IL/src/il_icon.c	2010-02-07 17:15:59.942840566 +0100
@@ -530,7 +530,11 @@
 
 	// Expand low-bit-depth grayscale images to 8 bits
 	if (color_type == PNG_COLOR_TYPE_GRAY && bit_depth < 8) {
+#if PNG_LIBPNG_VER < 10400
 		png_set_gray_1_2_4_to_8(ico_png_ptr);
+#else
+		png_set_expand_gray_1_2_4_to_8(ico_png_ptr);
+#endif
 	}
 
 	// Expand RGB images with transparency to full alpha channels
